app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="isListExpanded"
+      type="Boolean" />
+
+     <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.hintsandsolution.HintsViewModel" />
+  </data>
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="24dp"
+        android:layout_marginStart="72dp"
+        android:layout_marginEnd="72dp"
+        android:layout_marginBottom="20dp"
+        android:orientation="horizontal">
+
+        <TextView
+          android:id="@+id/hint_title"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="center"
+          android:layout_marginTop="8dp"
+          android:layout_marginBottom="8dp"
+          android:fontFamily="sans-serif-medium"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="20sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <Button
+          android:id="@+id/reveal_hint_button"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:minWidth="160dp"
+          android:minHeight="48dp"
+          android:layout_marginStart="8dp"
+          android:layout_marginTop="28dp"
```

Comment:
This should be `16dp` if above suggestions are done, so please calculate from your side what it should be.

Reply:
Done.